File: Service/FinCare/RMA.Service.FinCare.Contracts/Entities/Commissions/CommissionDetail.cs
using RMA.Common.Entities;

namespace RMA.Service.FinCare.Contracts.Entities.Commissions
{
    public class CommissionDetail : AuditDetails
    {
        public int DetailId { get; set; }
        public int HeaderId { get; set; }
        public int InvoicePaymentAllocationId { get; set; }
        public string InvoiceNumber { get; set; }
        public string PolicyNumber { get; set; }
        public string RepCode { get; set; }
        public string RepName { get; set; }
        public decimal AllocatedAmount { get; set; }
        public double CommissionPercentage { get; set; }
        public double AdminPercentage { get; set; }
        public string CommissionFormula { get; set; }
        public decimal CommissionAmount { get; set; }
        public string AdminServiceFeeFormula { get; set; }
        public decimal AdminServiceFeeAmount { get; set; }
        public decimal TotalAmount { get; set; }
        public CommissionInvoicePaymentAllocation InvoicePaymentAllocation { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/payment-manager/models/commission-detail.ts
import { CommissionInvoicePaymentAllocation } from './commission-invoice-payment-allocation';

export class CommissionDetail {
    detailId: number;
    commissionFormula: string;
    invoicePaymentAllocationId: number;
    invoiceNumber: string;
    policyNumber: string;
    repCode: string;
    repName: string;
    allocatedAmount: number;
    headerId: number;
    commissionPercentage: number;
    adminPercentage: number;
    commissionAmount: number;
    adminServiceFeeAmount: number;
    totalAmount: number;
    adminServiceFeeFormula: string;
    invoicePaymentAllocation:CommissionInvoicePaymentAllocation
}